pom.xml

```diff
@@ -123,6 +123,15 @@
             <artifactId>maven-invoker</artifactId>
             <version>3.0.1</version>
         </dependency>
+        <dependency>
+            <!--
+            The maven-dependency-plugin wants this to be listed explicitly as it comes as a transitive dependency
+            of maven-model.
```

Comment:
So that means another thing is using this dependency already? I'd rather not risk the version dependency clashes. Seems like you just need an empty string, a null check, and a long check to replace this dependency.

Reply:
As this plugin relies on `maven-model` (they all do) all plugin code has access to `commons-lang3`.

Reply:
...except now we have a dependency on the specific version of that dependency. That may be the same as maven-model now, but maybe not later.

Reply:
I'll address this.

Reply:
Removed the _use_ of the dependency.